README.md

```diff
@@ -5,178 +5,160 @@
     Templates for creating a Docker model submission on Synapse.
 </h3>
 
-You can either build off of this repository template or use it as reference 
-to build your model from scratch.  We have provided a sample model template
-for both R and Python.
+You can either build off of this repository template or use it as reference
+to build your model from scratch. Sample model templates for both R and
+Python are provided.
 
 ### Requirements
-* Python or R
-* [Docker](https://docs.docker.com/get-docker/)
-* [Synapse account](https://www.synapse.org/#)
-* Synapse project for the challenge
+
+- Python or R
+- [Docker](https://docs.docker.com/get-docker/)
+- [Synapse account](https://www.synapse.org/#)
+- Synapse project for the challenge
 
 ---
 
 ### Write your algorithm(s)
 
-1. Replace the code in the `run_model.*` script with your own algorithm(s).
-    You can create additional scripts for modularization/better organization
-    if desired.
+1. Replace the placeholder code in `run_model.*` script with your own
+   algorithm(s). Create additional functions and scripts for modularity
+   and organization as needed.
+
+2. Manage the dependencies:
+
+   - **Python:** Update `requirements.txt` with any required Python libraries.
+   - **R:** Update `requirements.R` by modifying the `pkg_list` variable to
+     include or exclude necessary R packages.
+
+3. (optional) Locally run `run_model.*` to verify it can run successfully.
+
+   The scripts have been designed to accept input and output directories as
+   command-line arguments, allowing you to test with various data locations.
 
-2. If using Python, update `requirements.txt` with any additional
-    libraries/packages used by your script(s).
+   - By default, the scripts look for input in the `/input` directory
+     and write output to the `/output` directory, as expected by the
+     Synapse submission system.
 
-    If using R, update `requirements.R` and add/remove any libraries/packages
-    listed in `pkg_list` that are used by your script(s).
+   - To use custom directories, specify them as arguments. For example:
 
-3. (optional) Locally run `run_model.*` to ensure it can run successfully.
+     **Python**
 
-    These scripts have been written so that the input and output files are not
-    hard-coded in the `/input` and `/output` directories, respectively (though
-    they are used by default).  This way, you can test your changes using any
-    directories as input and/or output.
+     ```
+     python run_model.py --input-dir ../sample_data/ --output-dir .
```

Comment:
It looks straightforward enough; 
Optionally, maybe adding a note clarifying that the command should be run from the `python/` or `r/` directory, Or just run from the root, like: `python python/run_model.py --input-dir sample_data/ --output-dir .`